### ISSUE TITLE
event showing on Legistar, included in Councilmatic email, but not on site

### ISSUE THREAD

**Issue by davidmooreppf** [datamade/nyc-councilmatic#203]
My morning email alert for "all meetings" included a link to upcoming event on Feb. 27 ::

http://nyc.councilmatic.org/event/committee-on-transportation-07e550614ab1/

But it's a Page Not Found, b/c event isn't yet listed here, most recent event is still Jan. 27th ::

https://nyc.councilmatic.org/committee/committee-on-transportation-5b69e06a3eda/

Feb. 27th event is indeed on Legistar ::

http://legistar.council.nyc.gov/DepartmentDetail.aspx?ID=6921&GUID=F01D07B8-2F8B-4A2D-836A-63FC030C1A34&Search=

... so is it being brought-in as expected? Meeting detail link :: 

http://legistar.council.nyc.gov/MeetingDetail.aspx?ID=528641&GUID=8FFBC651-45BC-41C3-960D-A19D201A4672&Options=&Search=

**Comment by davidmooreppf** [comment]
I should add, the other link in the email was a working link to an event, so they're certainly not all misfiring. 

https://nyc.councilmatic.org/event/committee-on-sanitation-and-solid-8493641f5e55/

**Comment by reginafcompton** [comment]
Well....we did not update our "crontab" for NYC, and so, the NYC site did not get updated data, nor did it send emails this morning.

The email you received actually came from the NYC-staging site. And the staging site has the meeting: 

https://nyc-staging.councilmatic.org/event/committee-on-transportation-07e550614ab1/

(The "test" emails have links that point to NYC, not the staging site. That way, we did not risk misdirecting people when we did the ol' merge-and-deploy.) 

@davidmooreppf, I fixed the crontab, and everything will fire as expected tomorrow morning. I'll keep this issue open, until then.
